doc/topics/windows/windows-package-manager.rst

```diff
@@ -817,30 +818,25 @@ Jinja:
         uninstall_flags: '/S'
       {% endfor %}
 
-In this example we are able to generate a software definition file that defines
-how to install 12 versions of Firefox. We use Jinja to create a list of
-available versions. That list is in a ``for loop`` where each version is placed
+In this example JINJA is used to to generate a package definition file that defines
+how to install 12 versions of Firefox. Jinja is used to create a list of
+available versions. The list is iterated through a  ``for loop`` where each version is placed
 in the ``version`` variable. The version is inserted everywhere there is a
 ``{{ version }}`` marker inside the ``for loop``.
 
-You'll notice that there is a single variable (``lang``) defined at the top of
-the software definition. Because these files are going through the Salt renderer
-many Salt modules are exposed via the ``salt`` keyword. In this case it is
-calling the ``config.get`` function to get a language setting that can be placed
-in the minion config. If it is not there, it defaults to ``en-US``.
+The single variable (``lang``) defined at the top of the package definition identifies the language of the package.
+You can access the Salt modules using the ``salt`` keyword. In this case, the ``config.get`` function is invokedto retrieve
+the language setting. If the ``lang`` variable is not defined then the default value is ``en-US``.
 
-Example: Latest
-===============
-
-There are some software vendors that do not provide access to all versions of
+Example: Package definition file to install the latest version
+===============================================================
+Some software vendors that do not provide access to all versions of
 their software. Instead they provide a single URL to what is always the latest
 version. In some cases the software keeps itself up to date. One example of this
-is the Google Chrome web browser.
-
-`Chrome <https://dl.google.com/edgedl/chrome/install/GoogleChromeStandaloneEnterprise.msi>`_
+is the `Chrome <https://dl.google.com/edgedl/chrome/install/GoogleChromeStandaloneEnterprise.msi>`_
```

Comment:
**Nitpick (grammar, non-blocking):** Can you add the full name to the link "Google Chrome web browser"? Also add a period at the end.

Reply:
Changed it to e `Google Chrome web browser <https://dl.google.com/edgedl/chrome/install/GoogleChromeStandaloneEnterprise.msi>`_.